janusgraph-core/pom.xml

```diff
@@ -64,6 +64,11 @@
             <artifactId>spatial4j</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.vividsolutions</groupId>
+            <artifactId>jts</artifactId>
```

Comment:
@pluradj Good catch.
@mbrukman We should have some license automation checks too.
@sjudeng Please let us know if there are any alternatives.